app/controllers/redirect/contact_controller.rb

```diff
@@ -2,7 +2,7 @@ module Redirect
   class ContactController < RedirectController
     def show
       redirect_to_and_log(
-        IdentityConfig.store.idv_contact_url,
```

Comment:
This configuration value is not assigned correctly in production, and is redundant with the `MarketingSite` class method. Luckily the controller isn't used currently.